java-manta-examples/src/main/java/ClientEncryptionRangeDownload.java

```diff
@@ -0,0 +1,59 @@
+import com.joyent.manta.client.MantaClient;
+import com.joyent.manta.client.MantaObjectResponse;
+import com.joyent.manta.config.*;
+import com.joyent.manta.http.MantaHttpHeaders;
+import sun.misc.BASE64Decoder;
```

Comment:
Avoid using any packages in the sun.* namespace. For base64 operations, use the java.util.Base64 class.